### ISSUE TITLE
Build warnings in Bob and Editor 2 (DEF-2917)

### ISSUE THREAD

**Issue by britzl** [defold/defold#3231]
*Description:*
It would be nice if the build process could output warnings, in both Bob and Editor 2.

*Use case:*
We recently modified the Collada parsers to allow invalid number entries in <float_arrays> (DEF-2902), but this is silent currently. It would be better if this was logged some way so that the user could see there is something wrong with the Collada file, but the invalid values was treated as zeroes.

*Business need:*
Better user experience.

**Comment by AGulev** [comment]
[Editor now shows logs from bob, for ](https://github.com/defold/defold/blob/1ae302ec33d4514408c04ad3ae5d3c1efe2057bd/com.dynamo.cr/com.dynamo.cr.bob/src/org/jagatoo/loaders/models/collada/stax/XMLFloat.java#L63)


https://github.com/defold/defold/blob/1ae302ec33d4514408c04ad3ae5d3c1efe2057bd/com.dynamo.cr/com.dynamo.cr.bob/src/org/jagatoo/loaders/models/collada/stax/XMLFloatArray.java#L67

**Comment by britzl** [comment]
@AGulev so we should be able to close this one right?

**Comment by AGulev** [comment]
I don't think so, it seems like we don't have warnings in places where we need them

**Comment by AGulev** [comment]
we are not going to fix it (collada is deprecated)
